### DISCUSSION TITLE
"ignorePaths" not working after enabling renovate from mend

### DISCUSSION THREAD

**Discussion by a-chatterjee** [renovatebot/renovate#18923] [Request Help]
This is my .whitesource file for renovate . The "ignorePaths" is not ignoring the files . Earlier I used a .renovate file with same setting (without line 2-6) and ignorePaths worked fine. Any insight on this is appreciated

```

1. {
2.   "checkRunSettings": {
3.       "vulnerableCheckRunConclusionLevel": "none"
4.   },
5.   "remediateSettings": {
6.       "enableRenovate": true,
7.       "extends": [
8.         "config:base",
9.         ":dependencyDashboard",
10.     ],
11.     "ignorePaths": [
12.         "**/node_modules/**",
13.         "pyproject.toml"
14.      ],
15.     "branchPrefix": "renovate-",
16.     "prHourlyLimit": 1,
17.     "packageRules":
18.     [
19.       {
20.         ......
21.         .......
22.       }
23.     ]
24.    }
25.   }

```

**Comment by rarkins** [comment]
I think this is related to `config:base` including the `:ignoreModulesAndTests` preset (https://docs.renovatebot.com/presets-default/#ignoremodulesandtests) and it is being resolved _after_ your `.whitesource` config so taking precedence.

Does it fix things if you add `"ignorePresets": [":ignoreModulesAndTests"]` to your config?

**Comment by a-chatterjee** [comment]
I added `"ignorePresets": [":ignoreModulesAndTests"]` . That didnt update the Detected dependencies in the `dependency dashboard`. It still shows the ignored dependencies listed.

This is how i added ignorePresets:
```

1. {
2.   "checkRunSettings": {
3.     "vulnerableCheckRunConclusionLevel": "none"
4.   },
5.   "remediateSettings": {
6.     "enableRenovate": true,
7.     "ignorePresets": [":ignoreModulesAndTests"],
8.     "extends": [
9.       "config:base",
10.       ":label(renovate)",
11.       ":dependencyDashboard",
12.       "group:monorepos"
13.     ],
14.     "ignorePaths": ["**/node_modules/**","pyproject.toml"],
15.     "branchPrefix": "renovate-",
16.     "prHourlyLimit": 1,
17.     "packageRules":
18.     [
19.       {
20.       .......
21.       }
22.     ],
23.     "schedule": [
24.       "every weekend"
25.     ]
26.    }
27.   }

```
Snippet of Dependency dashboard showing `pyproject.toml` after being ignored:
<img width="329" alt="Screenshot 2022-11-17 at 9 49 01 AM" src="https://user-images.githubusercontent.com/67306331/202354385-a638baff-e9de-4299-9ac5-03ee899f45f6.png">

**Reply by rarkins** [reply]
I can confirm in https://github.com/renovate-reproductions/18923 that a config similar to yours when running using the Renovate CLI does ignore the poetry file as expected.

I'm not sure why the same behavior isn't been seen when config is in the `.whitesource`. Could you state here or email me your repository name so that I can check the server side logs at Mend?

Also, did you know that you can still use a `renovate.json` file when running Mend? e.g. you could reduce your `.whitesource` to:

```

1. {
2.   "checkRunSettings": {
3.     "vulnerableCheckRunConclusionLevel": "none"
4.   },
5.   "remediateSettings": {
6.     "enableRenovate": true
7.    }
8.   }
```

And then let Renovate read from the `renovate.json` file in your repo. I only suggest this in case you'd like to try it for a quick solution.
